src/utils/hendelseFilteringUtils.tsx

```diff
@@ -33,6 +37,26 @@ export const filtrerPaaFodselsnummerEllerNavn = (personregister: PersonregisterS
 
 const getBirthDateFromFnr = (fnr: string): string => fnr.slice(0, 2);
 
+export const filterOnCompany = (personregister: PersonregisterState, companies: string[]) => {
+    if (!companies || !companies.length) {
```

Comment:
Skal det være `... || !companies.length === 0`?

Reply:
Bytter til `companies.length === 0` for å være litt tydeligere :)